### PR TITLE
Rename cheatcodes

### PR DESCRIPTION
**Pull request by Draggu**
## Introduced changes

- Rename cheatcodes

Follow-up for #2080

Next #2095

### CODE REVIEW

File: crates/forge/tests/integration/cheat_fork.rs [hunk 1]
```diff
@@ -53,42 +53,42 @@ fn prank_cairo0_contract() {
...
+                let uncheat_block_numbered_block_number = starknet::library_call_syscall(
                     CAIRO0_CLASS_HASH.try_into().unwrap(),
                     LIB_CALL_SELECTOR,
                     array![].span(),
                 ).unwrap_syscall()[0];
 
-                assert(*rolled_block_number == 123, 'start_roll does not work');
-                assert(unrolled_block_number == block_number, 'stop_roll does not work');
+                assert(*cheat_block_numbered_block_number == 123, 'does not work');
+                assert(uncheat_block_numbered_block_number == block_number, 'does not work');
```
Comment (drknzz) [inline crates/forge/tests/integration/cheat_fork.rs:91]: oof

File: crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/execution_info.rs [hunk 2]
```diff
@@ -159,7 +159,9 @@ pub fn get_cheated_exec_info_ptr(
         new_exec_info[2] = MaybeRelocatable::Int(
             (*cheated_data
                 .caller_address
-                .expect("No caller address value found for the pranked contract address")
+                .expect(
+                    "No caller address value found for the cheat_caller_addressed contract address",
```
Comment (drknzz) [inline crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/execution_info.rs:163]: 👁️👄👁️

File: snforge_std/src/cheatcodes.cairo [hunk 3]
```diff
@@ -212,16 +214,16 @@ fn stop_elect_global() {
...
+fn start_cheat_sequencer_address(target: ContractAddress, sequencer_address: ContractAddress) {
+    cheat_sequencer_address(target, sequencer_address, CheatSpan::Indefinite);
 }
 
 
-/// Cancels the `elect` / `start_elect` for the given target.
-/// - `target` - instance of `ContractAddress` specifying which contracts to stop electing
-fn stop_elect(target: ContractAddress) {
+/// Cancels the `cheat_sequencer_address` / `start_cheat_sequencer_address` for the given target.
+/// - `target` - instance of `ContractAddress` specifying which contracts to stop cheat_sequencer_addressing
```
Comment (drknzz) [inline snforge_std/src/cheatcodes.cairo:225]: `cheat_sequencer_addressing` yeah let's not 😄 
applies to other places too

File: snforge_std/src/cheatcodes.cairo [hunk 4]
```diff
@@ -26,10 +26,10 @@ fn test_address() -> ContractAddress {
 }
 
 /// Changes the block number for the given target and span.
-/// - `target` - instance of `ContractAddress` specifying which contracts to roll
+/// - `target` - instance of `ContractAddress` specifying which contracts to cheat_block_number
```
Comment (drknzz) [inline snforge_std/src/cheatcodes.cairo:29]: These comments are going to need some rewording if we want them to keep some sense

**Review by drknzz — COMMENTED** [review]
Unfortunately the renaming work will require stronger regular expressions when mass replacing D:

File: snforge_std/src/lib.cairo [hunk 5]
```diff
@@ -30,26 +30,26 @@ use cheatcodes::ContractAddress;
...
-use cheatcodes::warp_global;
-use cheatcodes::start_warp;
-use cheatcodes::stop_warp;
-use cheatcodes::stop_warp_global;
-use cheatcodes::elect;
-use cheatcodes::elect_global;
-use cheatcodes::start_elect;
-use cheatcodes::stop_elect;
-use cheatcodes::stop_elect_global;
+use cheatcodes::cheat_caller_address;
```
Comment (Arcticae) [inline snforge_std/src/lib.cairo:33]: If you don't want to update the docs here, there should be a follow up for this.
Please remember to update the docs and changelog

Comment (Draggu) [reply snforge_std/src/lib.cairo:33]: Docs will be in last PR

File: snforge_std/src/cheatcodes.cairo [hunk 6]
```diff
@@ -113,16 +113,16 @@ fn stop_prank_global() {
...
-    prank(target, caller_address, CheatSpan::Indefinite);
+fn start_cheat_caller_address(target: ContractAddress, caller_address: ContractAddress) {
+    cheat_caller_address(target, caller_address, CheatSpan::Indefinite);
 }
 
-/// Cancels the `prank` / `start_prank` for the given target.
-/// - `target` - instance of `ContractAddress` specifying which contracts to stop pranking
-fn stop_prank(target: ContractAddress) {
+/// Cancels the `cheat_caller_address` / `start_cheat_caller_address` for the given target.
+/// - `target` - instance of `ContractAddress` specifying which contracts to stop cheat_caller_addressing
```
[the remark this answers is not in the document]
Comment (Arcticae) [reply snforge_std/src/cheatcodes.cairo:124]: Also refers to all of the comments like this

File: snforge_std/src/cheatcodes.cairo [hunk 7]
```diff
@@ -30,10 +30,10 @@ fn test_address() -> ContractAddress {
 }
 
 /// Changes the block number for the given target and span.
-/// - `target` - instance of `ContractAddress` specifying which contracts to roll
+/// - `target` - instance of `ContractAddress` specifying which contracts to cheat_block_number
```
[the remark this answers is not in the document]
Comment (Draggu) [reply snforge_std/src/cheatcodes.cairo:33]: Fixed in next PR

File: snforge_std/src/cheatcodes.cairo (hunk 6 above)
[the remark this answers is not in the document]
Comment (Draggu) [reply snforge_std/src/cheatcodes.cairo:124]: Fixed in next PR

File: snforge_std/src/cheatcodes.cairo [hunk 8]
```diff
@@ -216,16 +218,16 @@ fn stop_elect_global() {
 }
 
 /// Changes the sequencer address for a given target.
-/// - `target` - instance of `ContractAddress` specifying which contracts to elect
+/// - `target` - instance of `ContractAddress` specifying which contracts to cheat_sequencer_address
```
Comment (drknzz) [inline snforge_std/src/cheatcodes.cairo:221]:
```suggestion
/// - `target` - instance of `ContractAddress` specifying which contract to cheat_sequencer_address
```
+ other places

Comment (drknzz) [inline snforge_std/src/cheatcodes.cairo:221]:
```suggestion
/// - `target` - instance of `ContractAddress` specifying which contract to apply the cheat to
```
it'd be easier to make this comment universal for other cheats too

**Review by drknzz — APPROVED** [review]

**Pull request merged**